pyproject.toml

```
@@ -7,7 +7,8 @@ name = "cpp-linter"
 description = "Run clang-format and clang-tidy on a batch of files."
 readme = "README.rst"
 keywords = ["clang", "clang-tools", "linter", "clang-tidy", "clang-format"]
-license = {text = "MIT License"}
+license = "MIT"
+license-files = [ "LICENSE" ]
```

We don't need to explicitly specify the `license-files` field. The LICENSE file is already automatically copied to the distribution. It should be enough to only specify the `license` field with the SPDX expression `"MIT"`.

```suggestion
license = "MIT"
```

It is my understanding that the  `license-files` field is meant for packages that need to distributed copies of multiple licenses.